index.js

```diff
@@ -18,43 +18,43 @@ function getNextUrl (linkHeader) {
 module.exports = (apiUrl, apiKey, options = {}) => {
   const log = options.log || (() => {})
 
-  const resolve = endpoint => {
-    const base2 = apiUrl.slice(-1) === '/' ? apiUrl.slice(0, -1) : apiUrl
-    const endpoint2 = endpoint.charAt(0) === '/' ? endpoint.slice(1) : endpoint
+  const canvasGot = got.extend({
+    headers: {
```

Comment:
I was thinking of moving the `baseUrl` into here, since we use if every time - whaddya say?

Reply:
Let's not - due to pagination logic.